spec/data_lint_spec.rb

```diff
@@ -47,4 +46,26 @@
       expect(document_types - allowed_document_types).to eql([])
     end
   end
+
+  describe "supergroups and subgroup document types" do
+    supergroups = YAML.load_file(File.dirname(__FILE__) + "/../data/supergroups.yml")
+    supertypes = GovukDocumentTypes::DATA
+
+    it "contains correct document types for both" do
```

Comment:
I'd split this into three tests, so the test descriptions can be explicit about what it is that's being tested.

Reply:
I can't see how this would be split up. What would each be testing?

Reply:
Had a conversation in meatspace with @fmlharrison about this.